.github/workflows/test_scala_2_12_non_spark.yaml

```
@@ -173,9 +193,65 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Install curl
+        run: |
+          apt-get update && apt-get install -y curl
+
+      - name: Run cloud aws tests
+        run: |
+          ./mill cloud_aws.test
+
+  online_tests:
```

actionlint (1.7.7)

key "online_tests" is duplicated in "jobs" section. previously defined at line:78,col:3. note that this key is case insensitive

(syntax-check)